From 24fe1f38bf9572606e2ec30f7222a9c227c1a284 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 29 Apr 2019 23:29:01 +0530
Subject: [PATCH 30/40] configs: update ls1043avnprdb.h for available PHYs

Avoid unused vitesse and realtek phy driver compilation.
Enable Fixed PHY driver support for SGMII connected to SJA1105 switch.
Enable support for AQR107 PHY at addr 0x1.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 include/configs/ls1043avnprdb.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/ls1043avnprdb.h b/include/configs/ls1043avnprdb.h
index 5717902148..b0417a8155 100755
--- a/include/configs/ls1043avnprdb.h
+++ b/include/configs/ls1043avnprdb.h
@@ -294,20 +294,21 @@
 
 
 #ifdef CONFIG_NET
-#define CONFIG_PHY_VITESSE
-#define CONFIG_PHY_REALTEK
 #define CONFIG_PHY_MICREL
 #define CONFIG_PHY_MICREL_KSZ9031
 #endif
 
 #ifdef CONFIG_SYS_DPAA_FMAN
 #define CONFIG_FMAN_ENET
+
+#define CONFIG_PHY_FIXED
 #define CONFIG_PHYLIB_10G
 #define CONFIG_PHY_AQUANTIA
 
 #define RGMII_PHY1_ADDR			0x1
 #define RGMII_PHY2_ADDR			0x2
 
+#define AQR107_PHY_ADDR1		0x1
 #define FM1_10GEC1_PHY_ADDR		0x1
 
 #define CONFIG_ETHPRIME			"FM1@DTSEC3"
-- 
2.17.1

